apps/emqx/src/emqx_logger_jsonfmt.erl

```diff
@@ -229,6 +229,13 @@ best_effort_json_obj(Map, Config) ->
             do_format_msg("~p", [Map], Config)
     end.
 
+json_value(true, _Config) ->
+    true;
+json_value(false, _Config) ->
+    false;
+json_value(V, Config) ->
+    json(V, Config).
+
 json(A, _) when is_atom(A) -> atom_to_binary(A, utf8);
```

Comment:
```suggestion
json(A, _) when is_atom(A) -> A;
```

Reply:
This works because all other atoms than true and false are converted to strings by the JSON library, right?

Reply:
Indeed, `jiffy` seems to encode atoms as JSON strings:

```elixir
iex(emqx@127.0.0.1)1> :jiffy.encode(%{a: :a})
"{\"a\":\"a\"}"
```

Reply:
Fixed: [e0d7a87](https://github.com/emqx/emqx/pull/12958/commits/e0d7a876ec4c92b800ae5bd9ff12e9cd1bc2e9a9)